### ISSUE TITLE
pstext sometimes fails when using -F+c option

### ISSUE THREAD

**Issue by evangowan** [GenericMappingTools/gmt#715]
I want to plot some text in the top left corner of a map, so I am using the -F+c option. However, I have found that sometimes it fails to plot the text. I have included a script to illustrate this problem. I tested this using GMT 5.4.5. I'm guessing this has to do with the -R option, but I don't know if there is a way around this.

```
#!/bin/bash

size="12p"
fontname="Helvetica-Bold"
color="black"
justification="+cTL" # the +c option plots relative to the corners of the map
text_angle="+a0"
text_options="-F+f${size},${fontname},${color} -F${justification} -F${text_angle} "

xshift=5
yshift=8

# fails to plot
plot="test1.ps"
pscoast -X${xshift} -Y${yshift} -R-78.5/61.4/-72/63r -JA-94/60/10c  -Df  -Bafg -P -A1000 -Wthinner -Slightgrey -K > ${plot}
pstext << END_TEXT  -O -J -R ${text_options} -V -P -Gwhite -D0.1/-0.25 >> ${plot}
test1
END_TEXT

# succeeds
plot="test2.ps"
pscoast -X${xshift} -Y${yshift} -R-101/54/-87.5/61.7r -JA-94/60/10c -Bafg  -Df   -P -A1000 -Wthinner -Slightgrey -K > ${plot}
pstext << END_TEXT  -O -J -R ${text_options} -V -P -Gwhite -D0.1/-0.25 >> ${plot}
test2
END_TEXT
```

**System information**

* Operating system: Ubuntu 16.04
* Version of GMT: 5.4.5

**Comment by seisman** [comment]
I can reproduce your issue with GMT 5.4.5, but it works with GMT 6.0.0 (not released yet). Maybe the issue has been fixed in the master branch.

As a workaround for GMT 5.4.5, you can add `-N` option to your `pstext` command, although I don't  understand why adding `-N` works.

**Comment by PaulWessel** [comment]
I have fixed this in 5.4 branch as well.  FYI the problem was roundoff of the coordinates for oblique projections and the points is outside the domain by a hair.
